### ISSUE TITLE
[json-routes] Make `method` param in JsonRoutes.add() case-insensitive

### ISSUE THREAD

**Issue by kahmali** [stubailo/meteor-rest#31]
Right now, the `method` param in the first arg of `JsonRoutes.add()` is case-sensitive, and must be passed as lowercase. Since it is very common to see these capitalized in the wild, we should accept any capitalization, and just `String.toLowercase()` whatever is passed. 

So both 
```javascript
JsonRoutes.add('get', 'widget', function (req, res, next) {});
```
and
```javascript
JsonRoutes.add('GET', 'widget', function (req, res, next) {});
```
would work.

If this is an acceptable change, I can put together a quick PR for it.

**Comment by stubailo** [comment]
This is a great idea, you should do it.

**Comment by kahmali** [comment]
Resolved in #35.
